### ISSUE TITLE
Video Slide

### ISSUE THREAD

**Issue by ostcar** [OpenSlides/OpenSlides#221]
Insert Video Slide that shows a video stream from extern source.

For example: live view of the speaker in a huge hall on several screens, so that everyone can see him; recorded video of a speach (Grußwort eines externen Obergurus) or a short movie, also advertise spot

**Comment by emanuelschuetze** [comment]
Add `iframe` tag to white list (idea: embed live stream into OpenSlides front page).

**Comment by ostcar** [comment]
see https://github.com/OpenSlides/openslides-conversations

**Comment by jsaalfeld** [comment]
The Plugin is something different from embedding a external video source (like twitch.tv or youtube.com stream). I don't know if it was possible with the 2.3 client to embed an iframe with some html-magic to load a video from an external source, but it could be added as an idea for the 3.0 projector.

**Comment by emanuelschuetze** [comment]
@jsaalfeld Right, this issue intended to create a possibility to embed external videos in a topic slide. With 2.3.x it's not allowed to embed iframes. We should discuss it for 3.0 client.

**Comment by tsiegleauq** [comment]
Yes Indeed. As it turned out, WebRTC is not the correct technology for embedding external streams.

TL;DR: WebRTC is (mostly exclusive) peer-to-peer.

Embedding a twitch, youtube, ffmpeg, data source into a html canvas could be sufficient for broadcasting an external speaker (given the stream was published somewhere else).

What my plugin does is allowing peer to peer chat and (video) calls, teamspeak/discord/mumble like VoIP and moderated discussions.

**Comment by emanuelschuetze** [comment]
see OpenSlides/openslides-client/issues/1927
